docs/setup/embedding.asciidoc

```diff
@@ -0,0 +1,49 @@
+[[embedding]]
+== Embedding {kib}
+
+You can embed a {kib} dashboard or visualization using an HTML code snippet generated with the *Copy iFrame code* button in *Share > Embed code* menu in *Dashboard* or *Visualize*.
```

Comment:
It would be good to provide some information about why you would want to embed a Kibana dashboard or visualization.  Can you also embed saved searches?

Reply:
> It would be good to provide some information about why you would want to embed a Kibana dashboard or visualization.

Sure, let me think how to phrase that properly.

> Can you also embed saved searches?

Technically you can embed anything. You just create an iframe and put any Kibana URL there assuming it works for your use case and you don't mind the risk of breaking it in any release.

I'd say we want to encourage our users to embed only what we officially support and provide advanced sharing controls for. But we also know that some users will still embed Kibana in very surprising ways. These are the users we're targeting in the section with `auth_provider_hint`.